specification/applicationinsights/resource-manager/readme.md

```diff
@@ -216,26 +216,24 @@ These settings apply only when `--tag=package-preview-2021-03-only` is specified
 input-file:
   - Microsoft.Insights/preview/2021-03-03-preview/diagnosticServicesToken_API.json
 ```
-### Tag: package-2020-02-12
 
-These settings apply only when `--tag=package-2020-02-12` is specified on the command line.
+### Tag: package-2020-10-only
 
-``` yaml $(tag) == 'package-2020-02-12'
+These settings apply only when `--tag=package-2020-10-only` is specified on the command line.
+
+```yaml $(tag) == 'package-2020-10-only'
 input-file:
-  - Microsoft.Insights/stable/2015-05-01/analyticsItems_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentAnnotations_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentApiKeys_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentContinuousExport_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentFeaturesAndPricing_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentProactiveDetection_API.json
-  - Microsoft.Insights/stable/2015-05-01/components_API.json
-  - Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json
-  - Microsoft.Insights/stable/2015-05-01/favorites_API.json
-  - Microsoft.Insights/stable/2015-05-01/webTestLocations_API.json
-  - Microsoft.Insights/stable/2015-05-01/webTests_API.json
   - Microsoft.Insights/stable/2020-10-20/myworkbooks_API.json
   - Microsoft.Insights/stable/2020-10-20/workbooks_API.json
-  - Microsoft.Insights/stable/2020-10-20/workbookOperations_API.json
+```
```

Comment:
Can we remove a tag from readme without noticing the corresponding service team? Does the service team know?
Go SDK always wants a tag that contains all functionalities in a RP to release, therefore if we are going to release appinsights for go SDK (currently no release request), we will have no candidates of tags to release.

Reply:
> Can we remove a tag from readme without noticing the corresponding service team? Does the service team know?
> Go SDK always wants a tag that contains all functionalities in a RP to release, therefore if we are going to release appinsights for go SDK (currently no release request), we will have no candidates of tags to release.

Yes, you are right. I remove this tag because it's never used and has been replaced by tag `package-2020-10-20` which contains all functionalities.

Reply:
Hi, I added "package-2020-02-12" with all the same contents of package-2020-10-20 because we got an S360 error due to that api-version being used and not defined in swagger. https://msazure.visualstudio.com/One/_workitems/edit/9298254

Reply:
That's why I remove the tag "package-2020-02-12" and add a new one "package-2020-10-20"